app/cdap/components/GridTable/index.tsx

```diff
@@ -206,9 +204,11 @@ export default function GridTable({ handleTransformationUpload, storeData }) {
         setShowBreadCrumb={setShowBreadCrumb}
         showBreadCrumb={showBreadCrumb}
         columnType={dataprep.types[selectedColumn]}
-        submitMenuOption={(option, datatype) =>
-          !transformationOptions.includes(option) ? onMenuOptionSelection(option, datatype) : null
-        }
+        submitMenuOption={(option, datatype) => {
+          !transformationOptions.includes(typeof option === 'object' ? option.value : option)
```

Comment:
use value variable 
and using if else condition set its value
use value variable in includes method